study-builder/studies/study-example.conf

```diff
@@ -73,6 +73,7 @@
   "studyDetails": [
```

Comment:
`studyDetails` is a different thing. We should change structure of `supportedLanguages` below.

Reply:
updated structure of supportedLanguages